root/static/scripts/common/utility/getCookie.js

```diff
@@ -6,7 +6,8 @@ import _cookies from './_cookies';
 let defaultExport = getCookieFallback;
 
 function getCookieFallback(name, defaultValue = undefined) {
-  return _cookies.hasOwnProperty(name) ? _cookies[name] : defaultValue;
+  let cookieHasName = Object.prototype.hasOwnProperty.call(_cookies, name);
+  return cookieHasName ? _cookies[name] : defaultValue;
```

Comment:
For this, it might be easier to just do:

```
  return Object.prototype.hasOwnProperty.call(_cookies, name)
    ? _cookies[name]
    : defaultValue;

```